src/PrestaShopBundle/Form/Admin/Type/MoneyWithSuffixType.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace PrestaShopBundle\Form\Admin\Type;
+
+use Symfony\Component\Form\Extension\Core\Type\MoneyType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
```

Comment:
I believe this form type already exists... wdyt @Quetzacoalt91 ?

Reply:
I'm pretty sure we already have this input type in Product page, isn't it?

Reply:
@mickaelandrieu yes I've seen this one, but it's just an extension to money type, it's extending my new type as well. It doesn't have the functionality to enter a text suffix right after the currency sign so that's why I've created a new type

Reply:
ok! fair enough then :+1: